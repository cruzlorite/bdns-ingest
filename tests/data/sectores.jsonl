{"descripcion": "Todos", "id": 24}
{"descripcion": "Cereales (parte I)", "id": 1}
{"descripcion": "Arroz (parte II)", "id": 2}
{"descripcion": "Azúcar (parte III)", "id": 3}
{"descripcion": "Forrajes desecados (parte IV)", "id": 4}
{"descripcion": "Semillas (parte V)", "id": 5}
{"descripcion": "Lúpulo (parte VI)", "id": 6}
{"descripcion": "Aceite de oliva y aceitunas de mesa (parte VII)", "id": 7}
{"descripcion": "Lino y cáñamo (parte VIII)", "id": 8}
{"descripcion": "Frutas y hortalizas (parte IX)", "id": 9}
{"descripcion": "Productos transformados a base de frutas y hortalizas (parte X)", "id": 10}
{"descripcion": "Plátanos (parte XI)", "id": 11}
{"descripcion": "Vino (parte XII)", "id": 12}
{"descripcion": "Árboles y otras plantas vivos, bulbos, raíces y similares, flores cortadas y follaje ornamental (parte XIII)", "id": 13}
{"descripcion": "Tabaco (parte XIV)", "id": 14}
{"descripcion": "Carne de vacuno (parte XV)", "id": 15}
{"descripcion": "Leche y productos lácteos (parte XVI)", "id": 16}
{"descripcion": "Carne de porcino (parte XVII)", "id": 17}
{"descripcion": "Carne de ovino y caprino (parte XVIII)", "id": 18}
{"descripcion": "Huevos (parte XIX)", "id": 19}
{"descripcion": "Carne de aves de corral (parte XX)", "id": 20}
{"descripcion": "Alcohol etílico de origen agrícola (parte XXI)", "id": 21}
{"descripcion": "Productos apícolas (parte XXII)", "id": 22}
{"descripcion": "Gusanos de seda (parte XXIII)", "id": 23}
