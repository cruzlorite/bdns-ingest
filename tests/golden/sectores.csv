id,descripcion,row_hash
24,Todos,5187582715518015649
1,Cereales (parte I),16716193514416091799
2,Arroz (parte II),10775018060622680495
3,"Azúcar (parte III)",10988020374656454572
4,Forrajes desecados (parte IV),976812500079969296
5,Semillas (parte V),13411510273067305267
6,"Lúpulo (parte VI)",12299599898027298766
7,Aceite de oliva y aceitunas de mesa (parte VII),15907518711729333683
8,"Lino y cáñamo (parte VIII)",13674766544083604976
9,Frutas y hortalizas (parte IX),5582015564762994417
10,Productos transformados a base de frutas y hortalizas (parte X),2639439135437094480
11,"Plátanos (parte XI)",13428371694039284596
12,Vino (parte XII),7776530619466849862
13,"Árboles y otras plantas vivos, bulbos, raíces y similares, flores cortadas y follaje ornamental (parte XIII)",7463656830267180446
14,Tabaco (parte XIV),13574765746625207658
15,Carne de vacuno (parte XV),18251896706720361536
16,"Leche y productos lácteos (parte XVI)",15034855017081439526
17,Carne de porcino (parte XVII),11325160760381521554
18,Carne de ovino y caprino (parte XVIII),7083865255668582329
19,Huevos (parte XIX),4206959392196537294
20,Carne de aves de corral (parte XX),1611956564523729242
21,"Alcohol etílico de origen agrícola (parte XXI)",4682845728671847152
22,"Productos apícolas (parte XXII)",7378307233766952153
23,Gusanos de seda (parte XXIII),3923931066054735144
